Mixpanel/Mixpanel.h

```diff
@@ -605,6 +633,27 @@ extern NSString *const MPNotificationTypeTakeover;
  */
 - (NSString *)libVersion;
 
+/*!
+ Opt out tracking.
+ 
+ This method is used to opt out tracking. This causes all events and people request no longer
+ to be sent back to the Mixpanel server.
+ */
+- (void)optOutTracking;
+
+/*!
+ Opt in tracking.
+ 
+ This method is used to opt in tracking. This causes all events and people request to be sent back
+ to the Mixpanel server.
+ */
+- (void)optInTracking;
```

Comment:
so we have parity with android what do you think if we create
`optInTrackingForDistinctId:(NSString *distinctId)` and `optInTrackingForDistinctId:(NSString *distinctId) withEventProperties:(NSDictionary *properties)` ?